Add note about Option<Vec<_>> to README.md (#10)
jplatte/serde_html_form: README.md

structs with fields of sequence type (e.g. `Vec<String>`). It also supports
`Option` in values, treating `foo=` as `foo: None`.

> For vectors that may be empty, it is best to use a `Vec` with the

Reviewer: ```suggestion
⚠️ For vectors that may be empty, it is best to use a `Vec` with the
```

> `#[serde(default)]` attribute, as the combination `Option<Vec<_>>` can lead
> to unexpected behavior when deserializing a `Vec` with exactly one value.

Reviewer: ```suggestion
`#[serde(default)]` attribute, as the combination `Option<Vec<_>>` will
often not work as expected.
```
